test/inductor/test_torchinductor_dynamic_shapes.py

```diff
@@ -972,15 +972,17 @@ def test_unspecialized_float_operations(self):
         for name, op in operations.items():
             with self.subTest(operation=name):
 
-                def fn(x, y):
-                    return op(x, y)
+                def fn(x, y, z):
+                    a = torch.exp(torch.tensor(z))
+                    return a * op(x, y)
 
                 cnt = CompileCounterWithBackend("inductor")
                 fn_opt = torch._dynamo.optimize(cnt)(fn)
 
                 x = torch.arange(3)
-                self.assertEqual(fn(x, 2.0), fn_opt(x, 2.0))
-                self.assertEqual(fn(x, 3.0), fn_opt(x, 3.0))
+                z = 1.3
+                self.assertEqual(fn(x, 2.0, z), fn_opt(x, 2.0, z))
+                self.assertEqual(fn(x, 3.0, z), fn_opt(x, 3.0, z))
                 self.assertEqual(cnt.frame_count, 1)
```

Comment:
It's up to your discretion, but you potentially might want to add this as a separate test, rather than overwriting the old one.